presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/BlockPositionValueState.java

```diff
@@ -15,12 +15,18 @@
 
 import com.facebook.presto.spi.block.Block;
 
-public interface UnknownBlockState
+/**
+ * Used for MinMaxBy states representing the value.
+ * The key should be in various types or null.
+ */
+public interface BlockPositionValueState
         extends TwoNullableValueState
 {
-    // First state is void and will always be null
```

Comment:
Shouldn't we keep this comment ? :)

Reply:
This is due to github diffing is wrong. These are two separate files.